admin_guide/scheduling/descheduler.adoc

```diff
@@ -0,0 +1,305 @@
+[[admin-guide-descheduler]]
+= Descheduling
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Descheculing involves evicting pods based on xref:admin-guide-descheduler-policies[specific policies] so that the pods can be rescheduled
+onto more appropriate nodes. 
+
+Your cluster can benefit from descheduling and rescheduling already-running pods for various reasons:
+
+* A nodes is under- or over-utilized.
+* Pod and node affinity requirements, such as tains or labels, have changed and the original scheduling decisions are longer appropriate for cetain nodes.
+* Node failure requires pods to be moved.
+* New nodes are added to clusters.
+
+Descheduler does not schedule replacement of evicted pods. The xref:../../admin_guide/scheduling/scheduler.adoc#admin-guide-scheduler[scheduler]
+automatically performs this task for the evicted pods.
+
+[NOTE]
+====
+The descheduler job is considered a _critical pod_, which prevents the descheduler pod from being evicted by the descheduler. 
+You can make a pod a critical pod by adding the the `scheduler.alpha.kubernetes.io/critical-pod` annotation to the pod specification 
+====
+
+The descheduler job and descheduler pod are created in `kube-system` project.
+
+[IMPORTANT]
+====
+The descheduler is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+The descheduler does not evict the following types of pods:
+
+* Critical pods (with the `scheduler.alpha.kubernetes.io/critical-pod` annotation).
+* Pods (link:https://kubernetes.io/docs/tasks/administer-cluster/static-pod/[static and mirror pods] or pods in standalone mode) not associated with a Replica Set (RS), Replication Controller (RC), Deployment, or Job (because these pods are not recreated).
```

Comment:
Do we support these features: static/mirror pods; standalone mode? Nothing in our docs about it.

Reply:
Should include deploymentconfig here?

Reply:
And do OpenShift doc have static and mirror pods? if have, better link to OpenShift instead if kubernetes.